app/main.py

```diff
@@ -81,6 +101,9 @@
         dcc.Store(id="store_live_alerts_data", storage_type="memory"),
         dcc.Store(id="last_displayed_event_id", storage_type="memory"),
         dcc.Store(id="images_url_current_alert", storage_type="session", data={}),
+
```

Comment:
Other dumb comment: should we remove the blank line ?

Reply:
Done!